Source/ACE.DatLoader/FileTypes/MotionTable.cs

```diff
@@ -146,10 +146,16 @@ public float GetAnimationLength(MotionStance stance, MotionCommand motion, Motio
         public float GetAnimationLength(AnimData anim)
         {
             var highFrame = anim.HighFrame;
+
+            // get the maximum # of animation frames
+            var animation = DatManager.PortalDat.ReadFromDat<Animation>(anim.AnimId);
+
             if (anim.HighFrame == -1)
+                highFrame = (int)animation.NumFrames;
+
+            if (highFrame > animation.NumFrames)
             {
-                // get the actual high frame from the animation length
-                var animation = DatManager.PortalDat.ReadFromDat<Animation>(anim.AnimId);
+                Console.WriteLine($"MotionTable.GetAnimationLength({anim}): highFrame({highFrame}) > animation.NumFrames({animation.NumFrames})");
```

Comment:
Is this a debug statement?

Reply:
yes, this is to see if it happens anywhere besides level 6 spell windup